package.json

```diff
@@ -160,6 +160,15 @@
               "image": "walkthrough/tutorials.png",
               "altText": "Tutorials"
             }
+          },
+          {
+            "id": "java.showHelpCenter",
+            "title": "Java Help Center",
+            "description": "Find more features and shortcuts in this page.\n[Open Java Help Center](command:java.welcome)",
```

Comment:
Discover more shortcuts, and features on this page ?

Reply:
ok, I'll take this.